internal_ws/yktrace/src/tir.rs

```diff
@@ -89,20 +89,20 @@ impl<'a, 'm> TirTrace<'a, 'm> {
             // terminator. Since the `do_not_trace` function can be recursive we count how many
             // times it is called and returned from and only stop ignoring things until we return
             // from the outer-most `do_not_trace` function.
-            if let Some(sym) = &dnt_trace_func {
+            if let Some(sym) = &dnt_func {
                 match &body.blocks[user_bb_idx_usize].term {
                     Terminator::Return => {
                         if sym == loc.symbol_name {
-                        dnt_trace_count -= 1;
-                        if dnt_trace_count == 0 {
-                            dnt_trace_func = None;
+                        dnt_count -= 1;
+                        if dnt_count == 0 {
+                            dnt_func = None;
                         }
                         }
```

Comment:
These two same-indent `}` suggest `cargo fmt` is needed?

Reply:
Fixed in 914b961.